_translator-files/po/de/Include-Server-Commands.po

```diff
@@ -3,128 +3,129 @@
-msgstr ""
+msgstr "`-d` oder `--discononquit` Trennt alle Clients beim Beenden. Wenn ein Server angehalten oder neu gestartet wird, werden normalerweise alle Clients, die ihre \"Disconnect\"-Schaltflächen nicht benutzt haben, die Verbindung wiederherstellen, wenn der Server wieder hochfährt. Mit dieser Option werden die Clients gezwungen, ihre Verbindungen zum Server manuell wiederherzustellen.\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:3
 #, no-wrap
 msgid "`-e`  or `--directoryserver`  Register the Server on a Directory (e.g. to set its genre (see also `-o`)). See [Server Types](#server-types) for further information.\n"
-msgstr ""
+msgstr "`-e` oder `--directoryserver` Registriert den Server in einem Verzeichnis (z.B. um sein Genre zu setzen (siehe auch `-o`)). Siehe [Server-Typen](#server-typen) für weitere Informationen.\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:4
 #, no-wrap
 msgid "`--directoryfile` Remember registered Servers even if the Directory is restarted. Directory Servers only. See [this guide](Custom-Directories) for further information. \n"
-msgstr ""
+msgstr "`--directoryfile` Registrierte Server speichern, auch wenn das Verzeichnis neu gestartet wird. Nur für Verzeichnis-Server. Siehe [dieses Handbuch](Benutzerdefinierte Verzeichnisse) für weitere Informationen\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:5
 #, no-wrap
 msgid "`-f`  or `--listfilter`     Whitelist Servers registering on the Server list, format `ip address 1[;ip address 2]` Directories only. \n"
-msgstr ""
+msgstr "`-f` oder `--listfilter` Whitelist Server, die in der Serverliste registriert sind, Format `ip address 1[;ip address 2]` Nur Verzeichnisse\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:6
 #, no-wrap
 msgid "`-F`  or `--fastupdate`     Reduces latency if Clients connect with \"Enable Small Network Buffers\" option. Requires faster CPU to avoid dropouts, and more bandwidth to enabled Clients.\n"
-msgstr ""
+msgstr "`-F` oder `--fastupdate` Verringert die Latenzzeit, wenn Clients mit der Option \"Kleine Netzwerkpuffer aktivieren\" verbunden sind. Erfordert eine schnellere CPU, um Aussetzer zu vermeiden, und mehr Bandbreite für aktivierte Clients.\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:7
 #, no-wrap
 msgid "`-l`  or `--log`            Enable logging, set path and file name                                                                    \n"
-msgstr ""
+msgstr "`-l` oder `--log` Protokollierung einschalten, Pfad und Dateiname festlegen\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:8
 #, no-wrap
 msgid "`-L`  or `--licence`        Show an agreement window before users can connect\n"
-msgstr ""
+msgstr "`-L`  or `—licence` Zeigt ein Zustimmungsfenster an, bevor Benutzer eine Verbindung herstellen können\n"
```

Comment:
```suggestion
msgstr "`-L`  or `--licence` Zeigt ein Zustimmungsfenster an, bevor Benutzer eine Verbindung herstellen können\n"
```
Using macOS/iOS autocorrect?

Reply:
Disabled all autocorrect and auto-replacing. 
Corrected